Cloud account - User invite feature added (#540)
OctopusDeploy/docs: docs/administration/authentication/octopusid-authentication.md

---

:::hint
Octopus ID authentication is available only in **Octopus Cloud** and is being progressively rolled out to customers.
:::

Octopus ID authentication allows you to log in to your Octopus Cloud instance using the same account that you use to sign in at [octopus.com](https://octopus.com). This allows you to manage who is able to access your Octopus instance from [your organization](https://octopus.com/organization/) and saves you time when moving between our website and your instance.

> ### Inviting Users & Configuring Teams with Octopus ID

Reviewer: ```suggestion
## Inviting Users & Configuring Teams with Octopus ID
```

> Once you've used octopus.com to [invite some other users](/docs/octopus-cloud/index.md#OctopusCloud-Invitingusers) to your instance, you are able to configure them with [Teams](../managing-users-and-teams/user-roles/teams-create-new.md) and [User Roles](../managing-users-and-teams/user-roles.md) as you normally would using the product.

Reviewer 2: Minor language tweak :)
```suggestion
After you've used octopus.com to [invite some other users](/docs/octopus-cloud/index.md#OctopusCloud-Invitingusers) to your instance, you can configure the users with [Teams](../managing-users-and-teams/user-roles/teams-create-new.md) and [User Roles](../managing-users-and-teams/user-roles.md) as you normally would using the product.
```
